### PR TITLE
Add filter options

### PR DESCRIPTION
**Pull request by fcheung**
Adds the ability to only dump a subset of schemas / and or tables - I think this will be useful later.

This obviously means that dropping db by default is not right, so I've added some control around that too.

### CODE REVIEW

File: lib/postgres_to_redshift.rb [hunk 1]
```diff
@@ -135,19 +156,31 @@ def schema_select_sql
...
       next if table.name =~ /^pg_/
       next if table.name =~ /^temp_/
       table.columns = column_definitions(table: table, schema: schema)
       table
     end.compact
+
+    if restrict_to_tables
+      restrict_to_tables & available_tables
+    else
+      restrict_to_tables
```
Comment (easonchan1213) [inline lib/postgres_to_redshift.rb]: shouldn't it be `available_tables`?

**Review by easonchan1213 — COMMENTED** [review]
Only one comment, otherwise looks good to me

File: lib/postgres_to_redshift.rb (hunk 1 above)
Comment (fcheung) [reply lib/postgres_to_redshift.rb]: Of course it should. thanks

**Comment by easonchan1213** [comment]
LGTM

**Comment by fcheung** [comment]
Thanks

**Pull request merged**